packages/developer-console-spa/src/components/consoleCodeBlock.module.css

```diff
@@ -0,0 +1,43 @@
+.sub-section {
```

Comment:
Can you rename the css files to follow the same convention as the other CSS files? i.e. `ConsoleCodeBlock.css` and `ConfigureSSI.css`

Reply:
Done